diffyscan/utils/common.py

```diff
@@ -57,3 +79,27 @@ def parse_repo_link(repo_link):
     repo_location = [item.strip("/") for item in parse_result[2].split("tree")]
     user_slash_repo = repo_location[0]
     return user_slash_repo
+
+
+def prettify_solidity(solidity_contract_content: str):
+    github_file_name = os.path.join(
+        tempfile.gettempdir(), "9B91E897-EA51-4FCC-8DAF-FCFF135A6963.sol"
```

Comment:
Can't we generate a random uuid here for the filename?

Reply:
done